src/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(NOVELRT_LIBRARY_HEADERS
   ../include/NovelRT/Utilities/Misc.h
   ../include/NovelRT/Windowing/WindowingService.h)
 
-  source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${NOVELRT_LIBRARY_HEADERS})
+  source_group(TREE ../ FILES ${NOVELRT_LIBRARY_HEADERS})
```

Comment:
I think you want ` ${CMAKE_CURRENT_SOURCE_DIR}/../`

Reply:
Ah ok, I did wonder. Thanks.